iteration.go

```diff
@@ -90,6 +90,48 @@ func (c *IterationController) Show(ctx *app.ShowIterationContext) error {
 	})
 }
 
+// Update runs the update action.
+func (c *IterationController) Update(ctx *app.UpdateIterationContext) error {
+	_, err := login.ContextIdentity(ctx)
```

Comment:
This is not required as you are not using the identity for anything below. The middleware will take care of unauthorized access.